STAR-492 side menu

oleynichenko opened the pull request:
![st1](https://user-images.githubusercontent.com/29056673/129018253-0efc5659-4353-4fae-9a0e-05f3c6823973.png)
![st2](https://user-images.githubusercontent.com/29056673/129018275-163a4375-6ac0-42a7-b1e1-4fc50ccd9fea.png)
![st3](https://user-images.githubusercontent.com/29056673/129184652-e7cd1466-ad7e-4c60-921a-66c6f1559043.png)

vitalii-retel commented on src/components/page/links.js:
```diff
@@ -0,0 +1,93 @@
+];
+
+const Links = ({ selectedEvent = {} }) => {
+  const links = useMemo(() => getLinks(selectedEvent.id), [selectedEvent]);
```
Dependency should by on `id`, I don't think useMemo checks the whole object, guess it checks only reference. But here we need `id` only.

oleynichenko replied:
fixed

vitalii-retel reviewed (commented):
- can we use em instead of px everywhere?
- please take a look at how we can test it already

oleynichenko commented:
> * can we use em instead of px everywhere?
fixed but not everywhere. For elements that do not depend on text I propose to keep px.

> * please take a look at how we can test it already
added for menu
